index.test-d.ts

```diff
@@ -0,0 +1,20 @@
+import logUpdate, {stderr, create} from '.';
```

Comment:
The named exports really bother me. They come with totally generic and useless names, which means everyone would have to rename them to `logUpdateStderr` or something. Why can't we just add them as properties on the default export?

Reply:
It's doable. do you want me to change that?

Reply:
Done.

Reply:
You have to update this import statement too then.

Reply:
I'm testing both kinds of imports.

Reply:
I don't want to support two different ways of using my modules.